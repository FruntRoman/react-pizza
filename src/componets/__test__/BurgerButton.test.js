import React from "react";
import BurgerButton from "../BurgerButton";
import renderer from "react-test-renderer";

describe('BurgerButton component:', () => {
    test('renders', () => {
        const component = renderer.create(
            <BurgerButton/>
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})