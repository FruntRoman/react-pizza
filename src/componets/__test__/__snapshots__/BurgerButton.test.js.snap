// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BurgerButton component: renders 1`] = `
<div
  className="wrap"
>
  <p
    className="burger-button button undefined"
  />
</div>
`;
